Project-Hotpot/Project-Hotpot/Parse-Backend/PFPlaylist.swift

```diff
@@ -57,18 +57,61 @@ class PFPlaylist: PFObject, PFSubclassing {
         }
     }
     
+    class func addPFSongToLastPlaylist(song:PFSong){
+        PFPlaylist.getAllPlaylistsInBackground {playlistArray, playlistError in
+            if playlistError == nil, let playlistArray = playlistArray {
+                // extracted PFPlaylist object successfully
+                // get last created playlist
+                let currPlaylist = playlistArray[0]
```

Comment:
There should be a more efficient way to do this. In SQL I would form a query like:

Select playlist from playlist_table
order by addedAt descending
limit 1

See if Parse allows something similar